qa/tasks/ses_qa.py

```diff
@@ -163,7 +173,10 @@ def drive_replace_check(self, **kwargs):
         Replaced osd_id should be back in the osd tree once stage.3 is ran
         """
         total_osds = self.master_remote.sh(number_of_osds_in_cluster)
-        assert int(total_osds) == 4, "Unexpected number of osds {} (expected 4)".format(total_osds)
+        drive_group = next(x for x in self.ctx['config']['tasks'] \
+                   if 'deepsea' in x and 'drive_group' in x['deepsea'])
+        disks = drive_group['deepsea']['drive_group']['custom']['data_devices']['limit']
```

Comment:
This is a code duplication, in order to improve code it is better to move it to a separate function, which will return an int value.

Reply:
Should be fixed now